tiledb/api/src/array/enumeration.rs

```diff
@@ -246,21 +247,42 @@ impl<'data> TryAsColumn<'data> for Enumeration<'data> {
 impl<'ctx> Debug for Enumeration<'ctx> {
     fn fmt(&self, f: &mut Formatter) -> FmtResult {
         let name = self.name().map_err(|_| fmt::Error)?;
-        let dtype = self
-            .datatype()
-            .map_err(|_| fmt::Error)?
-            .to_string()
-            .unwrap_or("<unknown datatype>".to_owned());
+        let dtype = self.datatype().map_err(|_| fmt::Error)?;
+
+        let dtype_string =
+            dtype.to_string().unwrap_or("<unknown datatype>".to_owned());
         let cell_val_num = self.cell_val_num().map_err(|_| fmt::Error)?;
         let ordered = self.ordered().map_err(|_| fmt::Error)?;
-
-        // TODO: Add enumeration values display.
+        let col = self.try_as_column().map_err(|_| fmt::Error)?;
+
+        let is_strings =
```

Comment:
Wouldn't `StringUcs4` need the offsets and have to do `to_boxed_u8_vec`?
Are attributes of that type sure to have `cell_val_num == u32::MAX`?